packages/react-avatar/src/Avatar.stories.tsx

```diff
@@ -189,19 +215,41 @@ export const ActiveAnimation = () => {
             icon={display === 'icon' ? <ContactIcon /> : undefined}
           />
         </div>
-        <Stack tokens={{ childrenGap: 8, maxWidth: 220 }}>
-          <PrimaryButton onClick={React.useCallback(() => setActive(a => !a), [])}>Toggle Active</PrimaryButton>
-          <SpinButton
-            label="activeDisplay"
-            value={activeDisplay}
-            onIncrement={nextActiveDisplay}
-            onDecrement={prevActiveDisplay}
-          />
-          <SpinButton label="display" value={display} onIncrement={nextDisplay} onDecrement={prevDisplay} />
-          <SpinButton label="size" value={`${size}`} onIncrement={nextSize} onDecrement={prevSize} />
-        </Stack>
-      </Stack>
-    </ThemeProvider>
+        <div className={flexStyles.stack}>
+          <Button onClick={React.useCallback(() => setActive(a => !a), [])}>Toggle Active</Button>
+          {/* Temporarly replacement of SpinButtons */}
+          <div className={flexStyles.flex}>
+            Active Display:
```

Comment:
It'd be helpful for this to display the current value of the prop:
```suggestion
            activeDisplay="{activeDisplay}"
```

Reply:
Agree on this as well, added it like this `<span>Active Display: {activeDisplay}</span>` as the Spin buttons are not used here anymore, do you think that's okay?

Reply:
True, I changed them to this -> `<span>activeDisplay="{activeDisplay}"</span>`